shell.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs ? import <nixpkgs> {},
+  target-os ? "" }:
+with pkgs;
+
+let
+  projectDeps = import ./default.nix { inherit target-os; };
+  targetMobile = {
```

Comment:
Same here, we should add `mobile` as an option. Maybe even `all` to both too for explicit building of everything?

Reply:
We could use all instead of "", yeah.

Reply:
Yeah, explicitly instead of implicitly. But we should keep `""` too.

Reply:
We don't really need `""`, since that's a fallback value we control.